suse_openstack_cloud

```diff
@@ -250,6 +250,16 @@ find_and_pconf_files /srv/tftpboot/discovery/pxelinux.cfg -type f
 find_and_pconf_files /srv/tftpboot/nodes                  -type f
 
 #############################################################
+section_header "Crowbar database"
+
+crowbar_db_dump_dir=/var/log/crowbar/db-dump
+# dump the database tables
+sqlite3 /opt/dell/crowbar_framework/db/production.sqlite3 .dump > $crowbar_db_dump_dir/production.dump.sql
```

Comment:
@MaximilianMeister just curious: could this also be done with rails ORM to be DB independent?